integrations/datadog/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,73 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: host
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"datadogHost"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            up: .up
+            hostName: .host_name
+            description: .description
+            metrics: .metrics
+            sources: .sources
```

Comment:
add 
- platform
- machine
- cpuCores

I would remove the `metrics` property, as we are not handling real time updates of host, so it won't be actually true data of cpu usage as it will only update by resync

Reply:
alright.